python/python/bystro/api/streaming.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from bystro.api.auth import authenticate
+
+GET_STREAM_ENDPOINT = "/api/jobs/{job_id}/streamFile"
+
+def stream_file(
+        job_id: str, output: bool = False, key_path: str | None = None, print_result: bool = True
+    ) -> None:
+    """
+    Fetch the file from the /api/jobs/:id/streamFile endpoint.
+
+    Parameters
+    ----------
+    job_id : str
+        The ID of the job being fetched.
+    output : bool, optional
+        Whether to fetch the output file (True) or the input file (False), by default False.
+    key_path : str, optional
+        The path to the desired file, required if `output` is True and it will direct to the output dir.
+        If `output` is False, `key_path` is used as an index into the input file dir. If not provided,
+        the first input file is used.
+    print_result : bool
+        Whether to print the result of the fetch operation, by default True.
+    """
+    if not job_id:
+        raise ValueError("Please specify a job id")
+
+    state, auth_header = authenticate()
+    url = state.url + GET_STREAM_ENDPOINT.format(job_id=job_id)
+
+    payload = {
+        "output": str(output),
```

Comment:
This doesn't work. You can't cast the boolean to string because the Bystro-api endpoint you wrote expects a boolean